torch_xla/csrc/runtime/pjrt_registry.cc

```diff
@@ -87,7 +87,7 @@ InitializePjRt(const std::string& device_type) {
              device_type == "ROCM") {
     TF_VLOG(1) << "Initializing PjRt GPU client...";
     bool async = sys_util::GetEnvBool(env::kEnvPjrtAsyncGpuClient, true);
-    int local_process_rank = sys_util::GetEnvInt(env::kEnvPjRtLocalRank, 0);
+    int local_process_rank = sys_util::GetEnvInt("LOCAL_RANK", 0);
```

Comment:
How do you feel about using a default value of -1 and then assert local_process_rank to be a non-negative, since we expect either kEnvPjRtLocalRank or LOCAL_RANK to be set and don't want to use a default value?

Reply:
I don't think this would work if you were prototyping in a notebook or REPL. Neither of these will be set automatically. `0` is a reasonable default value IMO.